tests/subsys/bluetooth/mesh/models/prj.conf

```diff
@@ -65,3 +65,5 @@ CONFIG_BT_MESH_LIGHT_SAT_SRV=y
 CONFIG_BT_MESH_LIGHT_HSL_SRV=y
 CONFIG_BT_MESH_LIGHT_HSL_CLI=y
 CONFIG_BT_MESH_SCHEDULER_CLI=y
+CONFIG_DM_MODULE=y
+CONFIG_BT_MESH_DM_CLI=y
```

Comment:
Shouldn't we disable legacy sensor APIs here?

Reply:
I forgot to add this to my latest push. Is disabling this still necessary?

Reply:
We should stop using soon to be deprecated legacy API.